src/components/CodeiumEditor/CompletionProvider.ts

```diff
@@ -59,14 +59,16 @@ export class MonacoCompletionProvider {
   constructor(
     grpcClient: PromiseClient<typeof LanguageServerService>,
     readonly setStatus: (status: Status) => void,
-    readonly setMessage: (message: string) => void
+    readonly setMessage: (message: string) => void,
+    readonly apiKey?: string | undefined
+
   ) {
     this.metadata = new Metadata({
       ideName: getBrowserVersion() ?? "unknown",
       ideVersion: getCurrentURL() ?? "unknown",
       extensionName: "@codeium/react-code-editor",
       extensionVersion: getPackageVersion() ?? "unknown",
-      apiKey: "d49954eb-cfba-4992-980f-d8fb37f0e942", // PLAYGROUND_API_KEY
+      apiKey: apiKey || "d49954eb-cfba-4992-980f-d8fb37f0e942", 
```

Comment:
I've been using `??` in these cases recently, doesn't really matter so up to you